### PR TITLE
CC-10508: enabled test.

### PR DESCRIPTION
**Pull request by abitskil**
- Enabled the tests for module.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -28,6 +28,7 @@ install:
 script:
     - find ./src -path src -prune -o -type f -name '*.php' -print0 | xargs -0 -n1 -P4 php -l -n | (! grep -v "No syntax errors detected" )
     - if [[ $DEFAULT == 1 ]]; then composer cs-check; fi
+    - if [[ $DEFAULT == 1 ]]; then composer test; fi
```
Comment (dereuromark) [inline .travis.yml:31]: please add tests as first run here.

**Comment by dereuromark** [comment]
Are you able to remove all codecept stuff and just use normal phpunit.xml etc?

**Comment by stereomon** [comment]
Can we close this?

**Comment by abitskil** [comment]
Yes, we can close it.

**Pull request closed without merging**